source/MaterialXView/Glsl/GLUtil/SimpleWindow.h

```diff
@@ -0,0 +1,102 @@
+#ifndef MATERIALX_SIMPLEWINDOW_H
+#define MATERIALX_SIMPLEWINDOW_H
+
+#include "HardwarePlatform.h"
+#include "WindowWrapper.h"
+
+namespace MaterialX
+{
+///
+/// @class SimpleWindow
+/// A platform independent window class. Plaftorm spefic resources
+/// are encapsulated using a WindowWrapper instance.
+///
+class SimpleWindow
```

Comment:
Window using platform dependent backend code. Is always offscreen and handles no events other than close.